packages/manageiq-ansible-venv/Dockerfile

```diff
@@ -14,27 +14,25 @@ ENV RPM_SPEC=manageiq-ansible-venv.spec \
 # CentOS repos: rpm-build and copr-cli dependencies
 # EPEL: copr-cli
 RUN sed -i 's/enabled=1/enabled=0/' /etc/yum/pluginconf.d/subscription-manager.conf
```

Comment:
@bdunne I noticed this line - should I make it match how we do it in the other images?

https://github.com/ManageIQ/manageiq-rpm_build/blob/8cc957a8dd8d2068335a64c65361156e3e2f72f8/Dockerfile#L8

Reply:
probably, the consistency would be nice